### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by zarva6596**
[DEMO LINK](https://zarva6596.github.io/react_carousel/)

### CODE REVIEW

**Review by mgrinko — APPROVED** [review]
It is a student approve :)

File: src/components/Carousel.js [hunk 1]
```diff
@@ -1,19 +1,213 @@
...
+
+    pos: 0,
+    img: 130,
+    step: 3,
+    animation: 1000,
+    frame: 3,
+  };
+
+  toRight = () => {
+    const list = document.querySelector('.Carousel__list');
```
Comment (Denchepornyuk) [inline src/components/Carousel.js:27]: не використовуй `document` в react

File: src/components/Carousel.js [hunk 2]
```diff
@@ -1,19 +1,213 @@
...
+          >
+            <ul
+              className="Carousel__list"
+              style={{
+                transition: `transform ${animation}ms`,
+              }}
+            >
+              {image.map(im => (
+                <li
+                  key={Math.random()}
```
Comment (Denchepornyuk) [inline src/components/Carousel.js:112]: для key не бажано використовувати random
https://medium.com/blackrock-engineering/5-common-mistakes-with-keys-in-react-b86e82020052

**Review by Denchepornyuk — CHANGES_REQUESTED** [review]

File: src/components/Carousel.js (hunk 2 above)
Comment (zarva6596) [reply src/components/Carousel.js:112]: ок поправлю

File: src/components/Carousel.js (hunk 1 above)
Comment (zarva6596) [reply src/components/Carousel.js]: готово

File: src/components/Carousel.js (hunk 2 above)
Comment (zarva6596) [reply src/components/Carousel.js]: поправив

**Review by Denchepornyuk — APPROVED** [review]

**Pull request closed without merging**